### ISSUE TITLE
WCAG error causes failing main branch

### ISSUE THREAD

**Issue by XanderGardner** [pulibrary/dpul#1337]
Failure needs to be fixed/excluded for main to pass tests and new ticket should be made if excluded.
```
Failures:

  1) accessibility when visiting catalog show page complies with WCAG
     Failure/Error:
       expect(page).to be_axe_clean
         .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
         .excluding(".tt-hint") # Issue is in typeahead.js library
         .skipping(:"color-contrast") # See issue: #1265
     
       Found 1 accessibility violation:
     
       1) duplicate-id: id attribute value must be unique (minor)
           https://dequeuniversity.com/rules/axe/4.3/duplicate-id?application=axeAPI
           The following 1 node violate this rule:
           
               Selector: iframe, #view > .intrinsic-container.intrinsic-container-16x9
               HTML: <div class="intrinsic-container intrinsic-container-16x9" id="viewer-container"><div id="uv" class="uv" style="width: 690px; height: 896px; display: none;"></div></div>
               Fix any of the following:
               - Document has multiple static elements with the same id attribute: viewer-container
               
       Invocation: axe.run({:exclude=>[[".tt-hint"]]}, {:runOnly=>{:type=>:tag, :values=>[:wcag2a, :wcag2aa, :wcag21a, :wcag21aa]}, :rules=>{:"color-contrast"=>{:enabled=>false}}}, callback);
     # ./spec/features/accessibility_spec.rb:183:in `block (3 levels) in <top (required)>'

  2) accessibility when visiting edit page complies with WCAG
     Failure/Error:
       expect(page).to be_axe_clean
         .according_to(:wcag2a, :wcag2aa, :wcag21a, :wcag21aa)
         .excluding(".tt-hint") # Issue is in typeahead.js library
         .skipping(:"color-contrast") # See issue: #1265
     
       Found 1 accessibility violation:
     
       1) duplicate-id: id attribute value must be unique (minor)
           https://dequeuniversity.com/rules/axe/4.3/duplicate-id?application=axeAPI
           The following 1 node violate this rule:
           
               Selector: iframe, #view > .intrinsic-container.intrinsic-container-16x9
               HTML: <div class="intrinsic-container intrinsic-container-16x9" id="viewer-container"><div id="uv" class="uv" style="width: 690px; height: 896px; display: none;"></div></div>
               Fix any of the following:
               - Document has multiple static elements with the same id attribute: viewer-container
               
       Invocation: axe.run({:exclude=>[[".tt-hint"]]}, {:runOnly=>{:type=>:tag, :values=>[:wcag2a, :wcag2aa, :wcag21a, :wcag21aa]}, :rules=>{:"color-contrast"=>{:enabled=>false}}}, callback);
     # ./spec/features/accessibility_spec.rb:328:in `block (3 levels) in <top (required)>'
```
